src/tls.rs

```diff
@@ -463,6 +463,26 @@ impl ServerCertVerifier for NoVerifier {
     }
 }
 
+/// Hyper extension carrying extra TLS layer information.
+/// Made available to clients on responses when `tls_info` is set.
+#[derive(Clone)]
+pub struct TlsInfo {
+    pub(crate) peer_certificate: Option<Vec<u8>>,
+}
+
+impl TlsInfo {
+    /// Get the DER encoded leaf certificate of the peer.
+    pub fn peer_certificate(&self) -> Option<&[u8]> {
```

Comment:
Curious, would the peer certificate always be in DER format? Or would different backends have it in different formats?

Reply:
DER is the binary wire format, so I would expect all TLS implementations to at some point be handling the DER-encoded binary representation of the server certificate / certificate chain.  Providing the binary representation seems to be consistent with how rustls and native-tls represent peer certificates.  Are you suggesting to change the type, or is this a comment about the doc string?